### PR TITLE
Creating login

### PR DESCRIPTION
**Pull request by girlpluscomputer**
ipad
iphone5
iphone6

### CODE REVIEW

File: src/components/Loader/Loader.js [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+      <div className="rect2" />
+      <div className="rect3" />
+      <div className="rect4" />
+      <div className="rect5" />
     </StyledLoader>
   );
-}
+};
 
 export default Loader;
```
Comment (arathjz) [inline src/components/Loader/Loader.js]: Line break here please

File: src/firebaseConfig.js [hunk 2]
```diff
@@ -4,18 +4,17 @@ const {
   REACT_APP_DATABASE_URL,
   REACT_APP_PROJECT_ID,
   REACT_APP_MESSAGING_SENDER_ID,
-  REACT_APP_APP_ID,
+  REACT_APP_APP_ID
```
Comment (arathjz) [inline src/firebaseConfig.js]: Missing comma here

Objects should be like


```
const object = {
  a: 1,
 b: 2,
}
```

Adding commas at the end of each key

File: src/firebaseConfig.js [hunk 3]
```diff
@@ -4,18 +4,17 @@ const {
...
 const firebaseConfig = {
   apiKey: REACT_APP_API_KEY,
   authDomain: REACT_APP_AUTH_DOMAIN,
   databaseURL: REACT_APP_DATABASE_URL,
   projectId: REACT_APP_PROJECT_ID,
-  storageBucket: "",
+  storageBucket: '',
   messagingSenderId: REACT_APP_MESSAGING_SENDER_ID,
-  appId: REACT_APP_APP_ID,
+  appId: REACT_APP_APP_ID
```
Comment (arathjz) [inline src/firebaseConfig.js]: Missing comma here

File: src/toast.js [hunk 4]
```diff
@@ -1,22 +1,19 @@
...
   return Toastify({
-    text: errorMessage,
+    text: message,
     duration,
     close: true,
-    gravity: "top",
+    gravity: 'top',
     position: 'right',
     backgroundColor,
     stopOnFocus: true
```
Comment (arathjz) [inline src/toast.js]: Missing comma

File: src/views/Login/Login.js [hunk 5]
```diff
@@ -1,7 +1,92 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import * as firebase from 'firebase/app';
+import 'firebase/auth';
 
-const Login = () => {
```
Comment (arathjz) [inline src/views/Login/Login.js]: there are too many line breaks here, it should be only one

File: src/views/Login/Login.js [hunk 6]
```diff
@@ -1,7 +1,92 @@
...
+    e.preventDefault();
+    authentication(email, password);
+  };
+
+  const handleChange = e => {
+    const {
+      target: { name, value }
+    } = e;
+
+    setData({ ...data, [name]: value });
```
Comment (arathjz) [inline src/views/Login/Login.js]: Greeeeat

Comment (girlpluscomputer) [reply src/views/Login/Login.js]: Thank you!

File: src/components/Loader/Loader.js (hunk 1 above)
Comment (girlpluscomputer) [reply src/components/Loader/Loader.js]: Done

File: src/firebaseConfig.js (hunk 2 above)
Comment (girlpluscomputer) [reply src/firebaseConfig.js]: Done

**Comment by girlpluscomputer** [comment]
> Well done!

Hooray!

**Pull request merged**